ntbs-service/DataMigration/CaseManagerImportService.cs

```diff
@@ -44,9 +44,12 @@ public class CaseManagerImportService : ICaseManagerImportService
             var ntbsCaseManager = existingCaseManager ??
                                   new User {IsActive = false, Username = legacyCaseManager.Username, CaseManagerTbServices = new List<CaseManagerTbService>()};
 
-            ntbsCaseManager.GivenName = legacyCaseManager.GivenName;
-            ntbsCaseManager.FamilyName = legacyCaseManager.FamilyName;
-            ntbsCaseManager.DisplayName = $"{ntbsCaseManager.GivenName} {ntbsCaseManager.FamilyName}";
+            if (!ntbsCaseManager.IsActive)
+            {
+                ntbsCaseManager.GivenName = legacyCaseManager.GivenName;
+                ntbsCaseManager.FamilyName = legacyCaseManager.FamilyName;
+                ntbsCaseManager.DisplayName = $"{ntbsCaseManager.GivenName} {ntbsCaseManager.FamilyName}";
```

Comment:
I think it would be safer to use `$"{legacyCaseManager.GivenName} {legacyCaseManager.FamilyName}"` here - e.g. if someone accidentally changed the order of these lines then the behaviour would change